File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
  int n;
    do
    {
        n = get_int("Height : ");
    }while(n<1 || n>8);
    for(int i = 0;i < n; i++)
    {
        for(int j=0;j<2*n+2;j++)
        {
            if(i+j<n-1 || j==n || j==n+1)
               printf(" ");
            else if(i+j >= n-1 && j-i <= n+2)
                printf("#");
         }
        printf("\n");
     }

}

File: mario.py
from cs50 import get_int
def find(height):
    for rows in range(1, height + 1):
        space = height - rows
        while (space > 0):
            print(" ",end ="")
            space = space - 1
        for has in range(1, rows +1):
            print("#",end = "")
        print("")
while (True):
    height = get_int("Height:")
   # x = int(height)
    if (height >= 1) and (height <= 8):
        find(height)
        break
    else:
        continue